modules/network-observability-RTT-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-RTT-overview_{context}"]
+= Round trip time tracing
+You can trace TCP handshake round trip time (RTT) to analyze network flows to help with determining application performance and network latency issues.
+
+When RTT tracing is enabled, you can see the following TCP handshake Round Trip Time with overall metrics represented in a donut or line chart in the *Overview*. 
+
+* Top X 90th percentile
```

Comment:
Similarly a mention of which panels are default and which arent is needed here

Reply:
similar to like something done in DNS docs https://github.com/openshift/openshift-docs/pull/68507/files#diff-aa7be6f7eeab2547efc5d05bb98aacafcb4fc5ca96d395736292b7f9761acc6aR22

Reply:
Got it. I re-worked this panels list.